docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Test Replay
+sidebar_position: 20
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What Test Replay is and why you should use it
+- How to get started with Test Replay in Cypress Cloud
+- How to debug failed and flaky CI test runs more accurately and efficiently
+
+:::
+
+## What Is Test Replay?
+
+Test Replay in Cypress Cloud allows developers to accurately and efficiently debug failed and flaky CI test runs. It captures every detail of test runs, enabling developers to replay and interact with the DOM, inspect network events, and console events as they happened during CI. This eliminates the need to replicate failed tests locally and enhances the ability to debug complex applications with confidence and speed.
+
+## Why Test Replay?
```

Comment:
wonder if a better title would be "Test Replay vs Screenshots and Videos"

Reply:
I can get down with that, I'll include as an h3 in this section.